bin/build_gem

```diff
@@ -0,0 +1,8 @@
+# Exit if any subcommand fails
+set -e
+
+VERSION=$(ruby -r ./lib/administrate/version.rb -e "puts Administrate::VERSION")
+yarn run build
+yarn run build:css
+gem build administrate.gemspec
+
```

Comment:
@pablobm Having this in a separate script is great!

I noticed the `VERSION` variable is not being used here. Are we missing the following line in this script?

```sh
gem install administrate-${VERSION}.gem
```

Reply:
Uh, I must have indeed :+1: Good catch.